Components/alert.css

```diff
@@ -0,0 +1,71 @@
+body {
+    width: 50%;
+    margin: 0 auto;
+    text-align: center;
+}
+
+.heading {
+    padding: 1.4rem 0;
+    font-size: 2rem;
+}
+
+.sub-heading {
+    font-size: 1.2rem;
+    margin-top: 2rem;
+}
+
+.info-text {
+    padding: 2rem 0;
+    font-size: 1rem;
+}
+
+.alert {
+    width: 50%;
+    color: var(--background-color);
+    border-radius: 4px;
+    background-color: var(--background-color);
+    padding: 1rem 1.5rem;
+    opacity: .9;
+    margin: 1rem auto;
+    font-size: .8rem;
+}
+
+.default {
+    background-color: var(--default-color);
+}
+
+.error {
```

Comment:
Since ```error``` classname is generic and you might have to use it in other components with different properties, you may use ```.alert.error``` as a class name here to increase specificity.

Reply:
Okay I'll change it to something else

Reply:
Done changed it to danger